content/nginx-one/_index.md

```diff
@@ -13,27 +13,52 @@ nd-product: NGINX One
 [//]: # "These are Markdown comments to guide you through document structure. Remove them as you go, as well as any unnecessary sections."
 [//]: # "Use underscores for _italics_, and double asterisks for **bold**."
 [//]: # "Backticks are for `monospace`, used sparingly and reserved mostly for executable names - they can cause formatting problems. Avoid them in tables: use italics instead."
-The F5 NGINX One Console makes it easy to manage NGINX instances across locations and environments. The console lets you monitor and control your NGINX fleet from one place—you can check configurations, track performance metrics, identify security vulnerabilities, manage SSL certificates, and more.
+F5 NGINX One Console makes it easy to manage NGINX instances across locations and environments. The console lets you monitor and control your NGINX fleet from one place—you can check configurations, track performance metrics, identify security vulnerabilities, manage SSL certificates, and more. NGINX One Console is part of NGINX One suite, which includes [additional components](#additional-components).
 
 ## Featured content
 [//]: # "You can add a maximum of three cards: any extra will not display."
 [//]: # "One card will take full width page: two will take half width each. Three will stack like an inverse pyramid."
 [//]: # "Some examples of content could be the latest release note, the most common install path, and a popular new feature."
 {{<card-layout>}}
   {{<card-section showAsCards="true" isFeaturedSection="true">}}
-    {{<card title="Get started" titleUrl="nginx-one/getting-started/" isFeatured="true" icon="unplug">}}
+    {{<card title="Get started" titleUrl="/nginx-one/getting-started/" isFeatured="true" icon="unplug">}}
       See benefits from the NGINX One Console
     {{</card >}}
-    {{<card title="Changelog" titleUrl="nginx-one/changelog/" icon="clock-alert">}}
+  {{</card-section>}}
+{{</card-layout>}}
+
+
+{{<card-layout>}}
+  {{<card-section showAsCards="true" >}}
+    {{<card title="Connect your NGINX instances" titleUrl="/nginx-one/connect-instances/" >}}
+      Work with data plane keys, containers, and proxy servers
+    {{</card>}}
+    {{<card title="Manage your NGINX instances" titleUrl="/nginx-one/nginx-configs/" >}}
+      Monitor and maintain your deployments
+    {{</card>}}
+    {{<card title="Organize users with RBAC" titleUrl="/nginx-one/rbac/" >}}
+      Assign responsibilities with role-based access control 
+    {{</card>}}
+    {{<card title="Automate with the NGINX One API" titleUrl="/nginx-one/api/" >}}
+      Manage your NGINX fleet over REST
+    {{</card>}}
+  {{</card-section>}}
+{{</card-layout>}}
+
```

Comment:
What if we listed everything else in the N1C docs here automatically, just including the top-level _index.md? Assuming the top-level directory represents the use case, we would get the same end result without having as much manual overhead.

We would just need to make sure the `title` we want to use is set in the frontmatter of the _index.md file for each directory (e.g., `title: Automate with the NGINX One API`). 

CC @lamATnginx

Reply:
> We would just need to make sure the title we want to use is set in the frontmatter of the _index.md file for each directory (e.g., title: Automate with the NGINX One API).

I would also want someplace for the extended descriptions, such as "Manage your fleet over REST", maybe in related `_index.md` files